requirements-status-messages.txt

```
@@ -0,0 +1 @@
+mattermostdriver
```

We'll want to pin to the latest version:

```suggestion
mattermostdriver==7.3.2
```